File: data/readable_format/fortran/file610.f90
module foodie_error_codes
  use, intrinsic :: iso_fortran_env, only : int32
  implicit none
  private
  public :: ERROR_UNSUPPORTED_SCHEME
  integer(int32), parameter :: ERROR_UNSUPPORTED_SCHEME = 1 
end module foodie_error_codes

program test_foodie_error_codes
  use foodie_error_codes
  implicit none

  ! Test the ERROR_UNSUPPORTED_SCHEME constant
  write(*,*) 'Testing ERROR_UNSUPPORTED_SCHEME...'
  if (ERROR_UNSUPPORTED_SCHEME == 1) then
    write(*,*) 'Test passed: ERROR_UNSUPPORTED_SCHEME = 1'
  else
    write(*,*) 'Test failed: ERROR_UNSUPPORTED_SCHEME /= 1'
  endif
end program test_foodie_error_codes

File: data/readable_format/cpp/file610.cpp
#include <iostream>
// Assuming the namespace foodie_error_codes is defined in this or another included file.
namespace foodie_error_codes {
    using I_P = int;
    constexpr I_P ERROR_UNSUPPORTED_SCHEME = 1;
}

int main() {
    std::cout << "Testing ERROR_UNSUPPORTED_SCHEME..." << std::endl;
    if (foodie_error_codes::ERROR_UNSUPPORTED_SCHEME == 1) {
        std::cout << "Test passed: ERROR_UNSUPPORTED_SCHEME = 1" << std::endl;
    } else {
        std::cout << "Test failed: ERROR_UNSUPPORTED_SCHEME /= 1" << std::endl;
    }

    return 0;
}